blockchain/integrity/integrity_test.go

```diff
@@ -684,48 +684,52 @@ func addTestingGetBlockHash(t *testing.T, hawaiiHeight uint64, bc blockchain.Blo
 
 	var (
 		zero     = big.NewInt(0)
+		nonce    = uint64(2)
 		gasLimit = testutil.TestGasLimit * 5
 		gasPrice = big.NewInt(testutil.TestGasPriceInt64)
+		bcHash   hash.Hash256
 	)
-
-	acHash2, err := addOneBlock(contract, 2, zero, gasLimit, gasPrice, getBlockHash(0)) // equal to all zero
-	require.NoError(err)
-	acHash3, err := addOneBlock(contract, 3, zero, gasLimit, gasPrice, getBlockHash(0)) // equal to block 2
-	require.NoError(err)
-	acHash4, err := addOneBlock(contract, 4, zero, gasLimit, gasPrice, getBlockHash(1)) // equal to block 2
-	require.NoError(err)
-	acHash5, err := addOneBlock(contract, 5, zero, gasLimit, gasPrice, getBlockHash(3)) // equal to block 1
-	require.NoError(err)
-	acHash6, err := addOneBlock(contract, 6, zero, gasLimit, gasPrice, getBlockHash(2)) // equal to block 3
-	require.NoError(err)
-	acHash7, err := addOneBlock(contract, 7, zero, gasLimit, gasPrice, getBlockHash(6)) // equal to genesis block
-	require.NoError(err)
-
 	tests := []struct {
-		acHash       hash.Hash256
-		commitHeight uint64
-		targetHeight uint64
+		commitHeight  uint64
+		getHashHeight uint64
 	}{
-		{acHash2, 2, 0},
-		{acHash3, 3, 2},
-		{acHash4, 4, 2},
-		{acHash5, 5, 1},
-		{acHash6, 6, 3},
-		{acHash7, 7, 0},
+		{2, 0},
+		{3, 5},
```

Comment:
b/c `commitHeight` is added one by one. So there is no need to be set in testdata